gradle.properties

```diff
@@ -27,3 +27,8 @@ android.enableJetifier=true
 android.defaults.buildfeatures.buildconfig=true
 android.nonTransitiveRClass=false
 android.nonFinalResIds=false
+
+# Enabling this property allows for setting a static version code that persist between builds,
+# enhancing Gradle caching efficiency and reducing unnecessary task runs, resulting in faster builds.
+# https://developer.android.com/build/optimize-your-build#use_static_build_properties
+useStaticVersionCode=false
```

Comment:
This should not be here. I mentioned user.properties file. It is not in the repo because it's used for local builds only. You don't need to include the property in any file in this PR, just adding the code that reads it. There are other examples of variables read from user.properties (`simultaneousDevProduction` for example) so check it out how they're read.

Reply:
Done.